libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -689,7 +689,7 @@ string YulUtilFunctions::storageArrayPopFunction(ArrayType const& _type)
 {
 	solAssert(_type.location() == DataLocation::Storage, "");
 	solAssert(_type.isDynamicallySized(), "");
-	solUnimplementedAssert(!_type.isByteArray(), "Byte Arrays not yet implemented!");
```

Comment:
Usually we just C++-call the other function in that case - otherwise we need checks at all places where we call this function.

Reply:
```
if (_type.isByteArray())
  return storageByteArrayPopFunction(_type);
```

Reply:
Cool, will do the same way